doc/source/merging.rst

```diff
@@ -504,7 +504,7 @@ the data in DataFrame.
 See the :ref:`cookbook<cookbook.merge>` for some advanced strategies.
 
 Users who are familiar with SQL but new to pandas might be interested in a
-:ref:`comparison with SQL<compare_with_sql.join>`.
+:ref:`comparison with SQL<F>`.
```

Comment:
What does "F" refer to?

Reply:
Oh, nothing. Looks like I added that error back when I was adding the `:ref:` entry for my new section. It will be reverted in my next push. Thanks!